src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(4,200px);
```

Comment:
```suggestion
  grid-template-columns: repeat(4, 200px);
```

Reply:
1 - you repeated this value several times
2 - it's a one of the main values on this page

create variable and reuse it